system-addon/content-src/message-center/message-format.md

```diff
@@ -0,0 +1,30 @@
+## Activity Stream Router message format
+
+Field name | Type     | Required | Description | Example / Note
+---        | ---      | ---      | ---         | ---
+`id`       | `string` | Yes | A unique identifier for the message that should not conflict with any other previous message
+`template` | `string` | Yes | An id matching an existing Activity Stream Router template | [See example](https://github.com/mozilla/activity-stream/blob/33669c67c2269078a6d3d6d324fb48175d98f634/system-addon/content-src/message-center/templates/SimpleSnippet.jsx)
+`publish_start` | `date` | Yes | When to start showing the message
+`publish_end` | `date` | Yes | When to stop showing the message
+`created` | `date` | Yes | When the message was created
+`modified` | `date` | No | When the message was last modified
+`content` | `object` | Yes | An object containing all variables/props to be rendered in the template. Values should not contain HTML (supporting a subset will be addressed in 62) | [See example below](#message-example)
+`campaign` | `string` | No | Campaign id that the message belongs to
+`targeting` | `string` `JEXL` | Yes | A [JEXL expression](http://normandy.readthedocs.io/en/latest/user/filter_expressions.html#jexl-basics) with all targeting information needed in order to decide if the message is shown | Not yet implemented
+
+### Message example
+```javascript
+{
+  id: "ONBOARDING_1",
+  template: "simple_snippet",
+  content: {
+    title: "Find it faster",
+    body: "Access all of your favorite search engines with a click. Search the whole Web or just one website from the search box.",
+    button: {
```

Comment:
leave the button part out for now, i'm not sure we're going to support nesting like this yet